src/mercury/boot/urls.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2018 Ruben Quinones (ruben.quinones@rackspace.com)
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+from mercury.boot.views import BootView, FileView
+
+
+boot_view = BootView.as_view('boot')
+file_view = FileView.as_view('file')
+
+
+boot_urls = [
+    # Boot url rules
+    ('/boot', boot_view),
+    ('/boot/<mac_address>', boot_view),
+    # File url rules
+    ('/file', file_view),
```

Comment:
What goes into a 'file'? Instructions on what to do during a reboot, like 'shutdown -r'?

Reply:
@jfwood it can be any file used to pxe boot a device, like an sfs file or an ipxe script.